example.py

```diff
@@ -44,9 +44,9 @@ def dummy_cruncher(tracks: Iterable[Track]) \
         )
 
         for start, end in intervals:
-            single_day, other = split_track_by_interval(track_a, start, end)
-            yield single_day, track_b, {"background": other,
-                                        "interval": (start, end)}
+            single_hour, other = split_track_by_interval(track_a, start, end)
+            yield single_hour, track_b, {"background": other,
+                                         "interval": (start, end)}
```

Comment:
This snippet makes total sense, starting at 5 o'clock at night no longer does. 

Though to be fair, that's not because of the diff in this PR; using `hours=1` on line 43 makes that odd.

Reply:
The `single_hour` is just an arbitrary and non-realistic selection. Plus, it is unclear what it means. Why not call it more descriptively `subtrack_for_interval`.